ecosystem/sep-0009.md

```diff
@@ -87,6 +87,14 @@ Name | Type | Description
 `organization.website` | string | Organization website
 `organization.email` | string | Organization contact email
 `organization.phone` | string | Organization contact phone
+`organization.individual_beneficial_owners.{index}.{SEP-0009 person field}` | array | Organization individual beneficial owners and control person
+`organization.entity_beneficial_owners.{index}.{SEP-0009 organization field}`| array | Organization entity beneficial owners
+`organization.individual_beneficial_owners.{index}.is_control_person` | boolean | Tells if an individual beneficial owner is also a control person
```

Comment:
I think there's some value in grouping all beneficial owners under a single group, then grouping persons and organizations under it. In the JSON it keeps them scoped to the one place and appears elegant, I think, although this is definitely personal preference bleeding into this request.
```suggestion
`organization.beneficial_owners.organizations.{index}.{SEP-0009 organization field}`| array | Organization entity beneficial owners
`organization.beneficial_owners.persons.{index}.is_control_person` | boolean | Tells if an individual beneficial owner is also a control person
```